modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterManager.java

```diff
@@ -268,29 +267,11 @@ public void localNodeElectedAsCoordinator() {
     }
 
     /**
-     * Gets the coordinator node's address. i.e address:port
-     *
-     * @return Address of the coordinator node
-     */
-    public String getCoordinatorNodeAddress() {
-        if (AndesContext.getInstance().isClusteringEnabled()) {
-            CoordinatorInformation coordinatorDetails = clusterAgent.getCoordinatorDetails();
-            String ipAddress = coordinatorDetails.getHostname();
-            String port = coordinatorDetails.getPort();
-            if (null != ipAddress && null != port) {
-                return ipAddress + "," + port;
-            }
-        }
-
-        return StringUtils.EMPTY;
-    }
-
-    /**
      * Gets address of all the members in the cluster. i.e address:port
      *
      * @return A list of address of the nodes in a cluster
      */
-    public List<String> getAllClusterNodeAddresses() {
+    public List<String> getAllClusterNodeAddresses() throws AndesException {
```

Comment:
Under method comment better to describe @return specifically mentioning what information is returned.